recipes-kernel/linux/files/bcm2712-raspberrypi5-xen.dts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2024 EPAM systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+*/
+#include "bcm2712-rpi-5-b.dts"
+
+/ {
+	chosen {
+		#address-cells = <0x2>;
+		#size-cells = <0x2>;
+		xen,xen-bootargs = "console=dtuart dtuart=/soc/serial@7d001000 dom0_mem=256M dom0_max_vcpus=1 xsm=flask flask=permissive";
```

Comment:
One comment - change dom0_mem=256M to dom0_mem=128M

Reply:
Ok